### ISSUE TITLE
flow.org/try throws an error when using 0.53.0

### ISSUE THREAD

**Issue by rsolomon** [facebook/flow#4626]
Empty block, or any code, will yield:

```
[LIB] static/v0.53.0/flowlib/bom.js:952:     body: ?ReadableStream,
                                                    ^ identifier `ReadableStream`. Could not resolve name
```

**Comment by jbrown215** [comment]
This is a KP, we'll have that fixed soon!

Thanks for reporting :)

**Comment by jbrown215** [comment]
This was fixed with ab721a590d887e8e261bdd248e856720eec32c1a
